### PR TITLE
Cohort Page

### PR DESCRIPTION
**Pull request by Mu7ammadAbed**

### CODE REVIEW

File: client/src/components/CohortPage/index.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            message: 'Error 401 Un-Authorized',
+            description: message,
+          });
+        }
+      },
+    });
+  };
+
+  // eslint-disable-next-line no-console
+  onClick = () => console.log('Clicked');
```
Comment (ranasobeid95) [inline client/src/components/CohortPage/index.js:66]:
```suggestion

```

File: client/src/components/CohortPage/index.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+  // eslint-disable-next-line no-console
```
Comment (ranasobeid95) [inline client/src/components/CohortPage/index.js:68]:
```suggestion
```

File: client/src/components/CohortPage/index.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  editCohort = (id) => console.log(`Edited ${id}`);
```
Comment (ranasobeid95) [inline client/src/components/CohortPage/index.js:69]: This function should redirect to ```Edit cohort page``` with cohort id as params.

File: client/src/components/CohortPage/index.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+class Cohort extends Component {
+  state = {
+    data: [],
+    startPage: 0,
+    endPage: 4,
+    total: 5,
+  };
+
+  async componentDidMount() {
+    const res = await axios.get('/api/v1/cohorts');
```
Comment (ranasobeid95) [inline client/src/components/CohortPage/index.js:20]: You need catch request errors with Axios so wrap it by ``` try-catch``` block

File: package.json [hunk 5]
```diff
@@ -30,6 +30,7 @@
   },
   "homepage": "https://github.com/GSG-G8/ca-wiki#readme",
   "dependencies": {
+    "@ant-design/icons": "^4.0.6",
```
Comment (MohammedAlghazali) [inline package.json:33]: please reomve this line, you add it in client/package.json

File: client/src/components/AdminCard/style.css [hunk 6]
```diff
@@ -47,19 +46,19 @@
...
-
-.admin-card a, .admin-card p {
+a, .admin-card p {
     color: var(--content-grey);
 }
+
+.edit {
+    color: var(--main-color);
+    padding: 4px 15px;
+}
```
Comment (ranasobeid95) [inline client/src/components/AdminCard/style.css:64]:
```suggestion
}

```

File: client/src/components/AdminContainer/style.css [hunk 7]
```diff
@@ -78,10 +73,35 @@
...
+  position: absolute;
+  top: 30%;
+  left: 40%;
+}
+
+.Add-Link {
+  background-color: #E8505B;
+  color: #FFFFFF;
+  width: 200px;
+  height: 58px;
```
Comment (ranasobeid95) [inline client/src/components/AdminContainer/style.css:99]: use css variables
```suggestion
 color: var(--main-white);
  width: 180px;
  height: 50px;
```

File: client/src/components/AdminContainer/index.js [hunk 8]
```diff
@@ -22,9 +17,11 @@ const AdminContainer = ({
           </div>
           <div className="admin-header-btn">
             {buttonContent !== undefined && (
-              <Button onClick={buttonFunction} type="primary" danger>
+              // <Button onClick={buttonRoute} type="primary" danger>
```
Comment (ranasobeid95) [inline client/src/components/AdminContainer/index.js:20]:
```suggestion
```

File: client/src/components/AdminContainer/style.css [hunk 9]
```diff
@@ -78,10 +73,35 @@
...
 }
 
+.pagination {
+  position: absolute;
+  left: 30%;
+  bottom: -10px;
+}
+.ant-modal {
+  margin-top: 130px;
+}
```
Comment (ranasobeid95) [inline client/src/components/AdminContainer/style.css:88]:
```suggestion
.ant-modal-wrap {
  margin-top: 80px;
}
```

**Pull request merged**